sdcm/cluster.py

```diff
@@ -3401,7 +3400,7 @@ def node_run_stress_bench(node, loader_idx, stress_cmd, node_list):
             log_file_name = os.path.join(logdir,
                                          'scylla-bench-l%s-%s.log' %
                                          (loader_idx, uuid.uuid4()))
-            ips = ",".join([n.private_ip_address for n in node_list])
```

Comment:
This is maybe correct, and we missed it, since no too many people run this in a multi-dc test.
it has nothing todo with running locally BTW, since this is running on the loader

Reply:
> also one general note,
> SCT_INTRA_NODE_COMM_PUBLIC was introduced for testing scylla-cloud, when the nodes are configured to use public_ip, and it's out of our control

> running with gce with
> `export SCT_IP_SSH_CONNECTIONS='public'` only should be enough in theory

It works well.